### ISSUE TITLE
Google doc to Bookdown/ Bookdown hosted on Gdrive

### ISSUE THREAD

**Issue by daaronr** [daaronr/dr-rstuff#8]
- can we get bookdowns hosted in Gdrive 
- and/or to export to decent Gdoc formats?

Note that 'cut and paste' from Rmd visual is actually pretty decent, in the interim

**Comment by oskasf** [comment]
@daaronr What do you mean by 'cut and paste' from Rmd?

The aim is to store bookdowns in Gdrive so that collaborators can preview them without having to link to a private github repo?

**Comment by daaronr** [comment]
Yes this was the main aim but there are also cases in which people want to leave intermediate feedback via google docs
